lib/apiservers/engine/backends/image.go

```diff
@@ -184,6 +155,7 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 		return err
 	}
 
+	layerCache.Update()
```

Comment:
No, that's the problem with imagec, remember? The docker persona knows absolutely nothing about an image it just pulled other than the reference that was used to pull it, and nowhere in the storage API is there support for getting a single layer based on this information.